packages/launcher-component/src/loaders/buildimage-loader.tsx

```diff
@@ -30,7 +30,12 @@ const convertToPairs = (object?: {[key: string]: string}): string[][] => {
 };
 
 export const convertToObject = (vars: string[][]) => {
-  return _.fromPairs(vars);
+  return _.fromPairs(vars.reduce<string[][]>((result, p) => {
```

Comment:
is it not possible to use `.filter` instead of `.reduce`?

Reply:
if you use filter instead of reduce you'll get `undefined` as an property name